src/theme/index.hbs

```diff
@@ -124,6 +124,7 @@
                                 <li role="none"><button role="menuitem" class="theme" id="coal">{{ theme_option "Coal" }}</button></li>
                                 <li role="none"><button role="menuitem" class="theme" id="navy">{{ theme_option "Navy" }}</button></li>
                                 <li role="none"><button role="menuitem" class="theme" id="ayu">{{ theme_option "Ayu" }}</button></li>
+				<li role="none"><button role="menuitem" class="theme" id="terminal">{{ theme_option "Terminal" }}</button></li>
```

Comment:
Indent the code
```suggestion
                                <li role="none"><button role="menuitem" class="theme" id="ayu">{{ theme_option "Ayu" }}</button></li>
								<li role="none"><button role="menuitem" class="theme" id="terminal">{{ theme_option "Terminal" }}</button></li>
```

Reply:
The indentation is all good, I couldn't commit the changes as the wasn't any. I've copy pasted your suggestion and there was absolutely no changes.
![image](https://user-images.githubusercontent.com/113247745/198852664-5e39c8b6-3385-4af5-ba43-2dcd6896bccf.png)